zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepo.java

```diff
@@ -46,28 +49,50 @@
    * @return
    * @throws IOException
    */
-  @ZeppelinApi public Note get(String noteId, AuthenticationInfo subject) throws IOException;
+  @ZeppelinApi
+  public Note get(String noteId, AuthenticationInfo subject) throws IOException {
+    Note note = getRaw(noteId, subject);
```

Comment:
You can put these logic in Note#postProcessParagraphs, so that you don't need to change the interface signature and lots of other things